backend/tests/README.md

```diff
@@ -0,0 +1,5 @@
+`cd /home/prakash/fast-music-remover`
+
+`PYTHONPATH=$(pwd) python -m unittest discover -s backend/tests -p 'test_*.py`
+
+Run the above two commands to test the code
```

Comment:
we should use pytest here, also as a side note, usually you would do:

```sh
cd /home/`whoami`/...
``` 

which would work for anyone using a Unix-based OS. 

Let's go with pytest regardless